include/sys/spa_impl.h

```diff
@@ -391,6 +391,8 @@ struct spa {
 	taskq_t		*spa_prefetch_taskq;	/* Taskq for prefetch threads */
 	uint64_t	spa_multihost;		/* multihost aware (mmp) */
 	mmp_thread_t	spa_mmp;		/* multihost mmp thread */
+	list_t		spa_leaf_list;		/* list of leaf vdevs */
+	vdev_t 		*mmp_last_leaf;  	/* last issued a write to */
```

Comment:
I don't believe we need to store the last mmp leaf in the same.  See comment below.